src/components/Message/MessageSimple/utils/renderText.tsx

```diff
@@ -1,9 +1,48 @@
 import React from 'react';
+// @ts-expect-error
```

Comment:
what's the error we're expecting? might want to add that description to the end so we know and could fix it in the future

Reply:
it wants us to import the @types for this lib, but it doesn't exist, i'll add a comment